mrtt-ui/src/components/MobileFooter/MobileFooter.js

```diff
@@ -28,18 +41,12 @@ const LinkContainer = styled(Link)`
   display: flex;
   flex-direction: column;
   align-items: center;
-  min-width: 80px;
-
-  color: ${(props) => (props.active === 'true' ? theme.color.primaryHover : theme.color.slub)};
-  div {
-    color: theme.color.slub;
+  color: ${(props) => (props.active === 'true' ? theme.color.primary : theme.color.text)};
+  @media (min-width: ${theme.layout.mediaQueryDesktop}) {
+    flex-direction: row;
+    gap: 1rem;
+    padding: ${themeMui.spacing(1)} ${themeMui.spacing(2)};
```

Comment:
Nice. I know its weird that we are using mystery MUI spacing, but importing themeMui directly lets us use way nicer syntax compared to pulling it out of function props.